docs/pages/installation.mdx

```diff
@@ -18,18 +18,21 @@ Teleport is officially supported on the platforms listed below. It is worth
 noting that the open-source community has been successful in building and
 running Teleport on UNIX variants other than Linux \[1].
 
-| Operating System | `teleport` Daemon | `tctl` Admin Tool | `tsh` User Client | Web UI (via the browser) | `tbot` Daemon |
+| Operating System | `teleport` Daemon | `tctl` Admin Tool | `tsh` and Teleport Connect User Clients [2] | Web UI (via the browser) | `tbot` Daemon |
 | - | - | - | - | - | - |
-| Linux v2.6.23+ (RHEL/CentOS 7+, Ubuntu 14.04+, and Debian 8+) \[2] | yes | yes | yes | yes | yes |
+| Linux v2.6.23+ (RHEL/CentOS 7+, Ubuntu 14.04+, and Debian 8+) \[3] | yes | yes | yes | yes | yes |
 | macOS v10.13+  (High Sierra)| yes | yes | yes | yes | yes |
-| Windows 10+ (rev. 1607) \[3] | no | no | yes | yes | no |
+| Windows 10+ (rev. 1607) \[4] | no | no | yes | yes | no |
 
 \[1] *Teleport is written in Go and it's possible to build it on
 any OS supported by the [Golang toolchain](https://github.com/golang/go/wiki/MinimumRequirements)*.
 
-\[2] *Enhanced Session Recording requires Linux kernel v5.8+*.
+\[2] *`tsh` is a Command Line Client (CLI) and Teleport Connect is an Graphical User Interface (GUI) desktop client. See
+  [Using Teleport Connect](connect-your-client/teleport-connect.mdx) for usage and installation*.
 
-\[3] *Teleport server does not run on Windows yet, but `tsh` (the Teleport client)
+\[3] *Enhanced Session Recording requires Linux kernel v5.8+*.
+
+\[4] *Teleport server does not run on Windows yet, but `tsh` (the Teleport client)
 supports most features on Windows 10 and later.*
```

Comment:
Should we mention Teleport Connect in this note?

Reply:
added!